### PR TITLE
Fixes Development environment check

### LINKED ISSUE #1098: Console spam after placing Charge Feeder Unit
Railcraft 10.0.1, Forge 12.18.3.2234, Infinity Lite 1.10 pack version 1.7.0.

I have a small electric track loop, and after I placed the IC2 charge feeder unit next to the track and supplied power and redstone, the server console is now full of "[Server thread/INFO] [railcraft]: Saving Charge Battery data..." about once every 45 seconds.  Not sure if it's intentional to have that message displayed in console or if I'm missing something obvious to filter it out, but I haven't seen any other mentions of this issue.

### PR DESCRIPTION
**Pull request by liach**
Fixes #1098
Normal game runs in Searge names, so `World` class is always present, causing issues.
Instead, field names are in `field_123456_a` forms, so getting a field by name may check if the game is running in a development environment, which, in theory, has fields deobfuscated.
Edited on GitHub since I am away from home.
Will wait for travis status.

### CODE REVIEW

**Review by CovertJaguar — CHANGES_REQUESTED** [review]
You should have changed OBFUSCATED, not DEV_ENV.

Any build tagged with "alpha" or "beta" is a development build.

**Comment by liach** [comment]
@CovertJaguar Do you still want this? Unfortunately, I do not have much time recently to work on this.

**Comment by liach** [comment]
@CovertJaguar Fixed

File: src/main/java/mods/railcraft/common/util/misc/Game.java [hunk 1]
```diff
@@ -27,11 +28,18 @@
...
     public static final boolean BUKKIT;
 
     static {
+        boolean dev = false;
+        try {
+            dev = Entity.class.getDeclaredField("worldObj") != null;
+        } catch (NoSuchFieldException | SecurityException ignored) {
+        }
+        OBFUSCATED = dev;
+        DEVELOPMENT_ENVIRONMENT = !Railcraft.getVersion().matches(".*(alpha|beta).*") || !OBFUSCATED;
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/common/util/misc/Game.java]: Huh....is it just me or this logic wrong? Not alpha or beta?

**Review by CovertJaguar — APPROVED** [review]
Looks good.

**Pull request merged**